src/components/snackbar/Snackbar.tsx

```diff
@@ -37,19 +38,64 @@ export type SnackbarProps = {
   customVisibleDurationMS?: number;
 };
 
-export const Snackbar = ({
+// if position is ever toggled, it starts using two SnackbarInstances, one for top and one for bottom
+// one of them is always disabled
+export const Snackbar = (snackbarProps: SnackbarProps) => (
+  <>
+    <SnackbarInstanceAtPosition snackbarProps={snackbarProps} position="top" />
+    <SnackbarInstanceAtPosition
+      snackbarProps={snackbarProps}
+      position="bottom"
+    />
+  </>
+);
+
+const SnackbarInstanceAtPosition = ({
+  snackbarProps,
+  position,
+}: {
+  snackbarProps: SnackbarProps;
+  position: SnackbarPosition;
+}) => {
+  // isDisabled means whether the SnackbarInstance should move to the hidden location
+  const isDisabled = snackbarProps.position !== position;
+  // isActivated means whether the SnackbarInstance component should exist
+  const [isActivated, setIsActivated] = useState(!isDisabled);
+  // the SnackbarInstance doesn't need to exist in its position until the position matches
+  // however don't revert it again, as it would ruin the exit animation
+  useEffect(() => {
+    if (!isDisabled) setIsActivated(true);
+  }, [isDisabled]);
```

Comment:
Is this really needed? I suspect the performance necessary for the `useEffect` and extra render outweighs the performance benefit of this code.

Reply:
Not really needed, just me thinking it would improve performance.
So I can remove it if you think it's not worth including?

Reply:
Removed here https://github.com/AtB-AS/mittatb-app/pull/4561/commits/0ad33acf777e4d35465d2d8354d65a653d233132